### PR TITLE
Remove sortable table ::before style, which set the # column

### PR DESCRIPTION
**Pull request by xybrnet**

### CODE REVIEW

File: umibukela/static/stylesheets/site.scss [hunk 1]
```diff
@@ -420,32 +420,7 @@ img.img-rounded-border {
 /* add a number column at the beginning of sortable tables, that can't itself be sorted and doesn't change on sorting */
 table.sortable tbody {
   counter-reset: sortabletablescope;
-	background-color: $white;
```
Comment (longhotsummer) [inline umibukela/static/stylesheets/site.scss:423]: woah, this is some interesting CSS. I don't know you could do this.

File: umibukela/static/stylesheets/site.scss [hunk 2]
```diff
@@ -610,10 +585,10 @@ footer.footer{
 		padding: 5px;
 		border-radius: 3px;
 		h1, h2 { margin: 0; color: $white ;fa}
```
Comment (longhotsummer) [inline umibukela/static/stylesheets/site.scss:587]: What's up with "fa" ?

**Comment by longhotsummer** [comment]
:+1:

File: umibukela/static/stylesheets/site.scss (hunk 1 above)
Comment (xybrnet) [inline umibukela/static/stylesheets/site.scss:423]: Yeah, me neither.

File: umibukela/static/stylesheets/site.scss (hunk 2 above)
Comment (xybrnet) [inline umibukela/static/stylesheets/site.scss:587]: Not sure either. It may be something with font-awesome. Though it almost seems more like typo.

**Pull request merged**